pkg/cloudproviders/aws/cloudprovider/pricing_test.go

```diff
@@ -175,4 +176,28 @@ var _ = Describe("Pricing", func() {
 		_, ok := p.SpotPrice("c99.large", "test-zone-1b")
 		Expect(ok).To(BeFalse())
 	})
+	It("should query for  `Linux/UNIX` and `Linux/UNIX (Amazon VPC)`", func() {
```

Comment:
Can you drop in a comment about what case this covers (e.g. EC2 Classic)
```suggestion
	It("should query for `Linux/UNIX` and `Linux/UNIX (Amazon VPC)`", func() {
```